examples/scripts/chat.py

```diff
@@ -0,0 +1,302 @@
+import os
+from threading import Thread
+from transformers import AutoModelForCausalLM, AutoTokenizer, TextIteratorStreamer, HfArgumentParser
```

Comment:
I think you can first import here `zero_verbose_init` from `trl.command.cli_utils` to remove all verbose and make sure the interface is clear at init

Reply:
see: https://github.com/huggingface/trl/blob/6cfa5cfc817be0f8acf30f8158a4a0ea244dc988/examples/scripts/sft.py#L55

Reply:
oh yes, fixed